index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav>
+                <a href="#">
+                    <img class="my-logo" src="images/my-logo.png" alt="logo">
+                </a>
+                <a href="#">
+                    <img class="menu" src="images/union.png" alt="union">
+                </a>
+            </nav>
+        </header>
+        <section class="main-section">
+            <br>
+            <h1 class="main-text">I’m Godanah
+                Glad to see you!</h1>
+            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <br>
+            <p>LET'S CONNECT</p>
+            <br>
+            <div class="contact">
+                <ul class="list">
+                    <li><a href="#"><img src="images/vector-1.png" alt="vector1"></a></li>
+                    <li><a href="#"><img src="images/vector-2.png" alt="vector2"></a></li>
+                    <li><a href="#"><img src="images/vector-3.png" alt="vector3"></a></li>
+                    <li><a href="#"><img src="images/vector-4.png" alt="vector4"></a></li>
+                    <li><a href="#"><img src="images/vector-5.png" alt="vector5"></a></li>
+                </ul>
+            </div>
+        </section>
+    </div>
+
+    <div class="card">
```

Comment:
- [ ] This being the container for the projects, add a `section` tags instead of using a `div` tag